src/queries/user/index.ts

```diff
@@ -109,7 +111,12 @@ const user: {
     info: (root) => root,
     // ipnsAddress,
     wallet: (root) => root,
-    settings: (root) => root,
+    settings: (root) => {
+      if (!root.id) {
```

Comment:
seems unnecessary since it's already protected by `settings: UserSettings! @auth(mode:"${AUTH_MODE.oauth}")`?

Reply:
`oauth` allow visitors querying protected fields on `viewer`.